Snowy-Path/Assets/SpawnPlayerPosition.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpawnPlayerPosition : MonoBehaviour
```

Comment:
Why is it empty ? What is it's purpose ?

Reply:
It's an object that i search to spawn the player, It was a rushing solution. Will need to change with a FindGameObjectsWithTag